spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -590,9 +590,38 @@
     expect(filter.to_ruby).to eq('<value ref=vm, type=integer>/virtual/used_disk_storage</value> >= 1048576000')
   end
 
+  it 'should test .is_numeric and .is_integer' do
+    expect(MiqExpression.is_numeric?('10,000.55')).to eq(true)
+    expect(MiqExpression.is_numeric?('10 000.55')).to eq(false)
+    expect(MiqExpression.is_numeric?('2,555.hello')).to eq(false)
+    expect(MiqExpression.is_numeric?('2,555.kilobytes')).to eq(true)
+    expect(MiqExpression.is_numeric?('2,555.55.megabytes')).to eq(true)
+
+    expect(MiqExpression.is_integer?('2,555')).to eq(true)
+    expect(MiqExpression.is_numeric?('2 555')).to eq(false)
+    expect(MiqExpression.is_integer?('2,555.kilobytes')).to eq(true)
+    expect(MiqExpression.is_integer?('2,555.hello')).to eq(false)
+  end
+
+  it 'should test .is_plural' do
+    _field = 'ManageIQ::Providers::InfraManager::Vm.storage-region_description' # vm belong_to storage
```

Comment:
Typically you'll use the underscore to designate an unused variable. Since you are using it, you can call this `field`